src/components/organisms/curriculum/OakRadioAsButton/OakRadioAsButton.test.tsx

```diff
@@ -226,12 +243,12 @@ describe("OakRadioAsButton", () => {
       </>,
     );
 
-    const radioGroup = container.querySelector('[role="radiogroup"]');
-    expect(radioGroup).toHaveAttribute("aria-labelledby", "group-label");
+    const radioButton = getByRole("radio");
+    expect(radioButton).toHaveAttribute("aria-labelledby", "group-label");
```

Comment:
This test is also a bit messy - it's a passing test that proves what it says it proves, but the scenario is sort of nonsensical because you would never label an individual radio button by the label of its group.